package.json

```diff
@@ -73,6 +73,7 @@
     "pg": "4.1.1"
   },
   "devDependencies": {
+    "async": "2.0.0-rc.3",
```

Comment:
Is it possible to drop this? I think the [sequence util](https://github.com/TryGhost/Ghost/blob/master/core/server/utils/sequence.js) does the same as  `async.waterfall` 😁